src/test-gui/java/de/qabel/desktop/ui/actionlog/ActionlogGuiTest.java

```diff
@@ -57,16 +73,21 @@ public void multilineInput() {
             robot.release(KeyCode.SHIFT);
         }
         robot.write("line2");
+    }
 
-        assertTrue(receiveMessages().isEmpty());
-        robot.push(KeyCode.ENTER);
-        assertFalse(receiveMessages().isEmpty());
-
-        DropMessage message = receiveMessages().get(0);
-        assertEquals(DropMessageRepository.PAYLOAD_TYPE_MESSAGE, message.getDropPayloadType());
-        assertEquals("line1\nline2", TextMessage.fromJson(message.getDropPayload()).getText());
+    private void writeTwoLinesOfText(String line1, String line2) {
```

Comment:
unused. Either remove it or use it instead of writeTwoLinesOfText. I would prefer the latter because it increases the tests readability / clearifies the intent:
```JAVA
writeTwoLinesOfText("line1", "line2");
...
assertEquals("line1\nline2", ...);
```